lib/datadog/ci/contrib/selenium/rum.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "datadog/tracing/contrib/patcher"
+
+require_relative "ext"
+require_relative "../../ext/test"
+
+module Datadog
+  module CI
+    module Contrib
+      module Selenium
+        # Provides functionality to interact with Datadog Real User Monitoring product
+        # via executing JavaScript code in the browser.
+        #
+        # Relevant docs: https://docs.datadoghq.com/real_user_monitoring/browser/
+        module RUM
+          def self.is_rum_active?(script_executor)
+            is_rum_active_script_result = script_executor.execute_script(Ext::SCRIPT_IS_RUM_ACTIVE)
+            Datadog.logger.debug { "[Selenium] SCRIPT_IS_RUM_ACTIVE result is #{is_rum_active_script_result.inspect}" }
+            is_rum_active_script_result == "true" || is_rum_active_script_result == true
```

Comment:
Should the `script_executor` be ensuring its result's a bool, rather than checking for both here?

Reply:
script_executor is coming from selenium-webdriver, it could be `Selenium::WebDriver::Manager` or `Selenium::WebDriver::Driver`

One could say that we need to be exactly sure what type this returns but in Ruby case one can never be sure. Another more beatiful way to write it would be `is_rum_active_script_result.to_s == "true"`, I think this would look better

Reply:
hehe, I would be more likely to try to cast the string to a bool :) but anyway, it's not very important, I think the code's fine as is.